src/scales/scale.time.js

```diff
@@ -830,9 +825,8 @@ export default class TimeScale extends Scale {
 	/**
 	 * @param {number} exampleTime
 	 * @return {number}
-	 * @private
 	 */
-	_getLabelCapacity(exampleTime) {
+	getLabelCapacity(exampleTime) {
```

Comment:
This is only used in this file. Can we keep it private?

Reply:
Done, using @ts-ignore